cmd/harmony/main.go

```diff
@@ -425,7 +425,7 @@ func setupConsensusAndNode(nodeConfig *nodeconfig.ConfigType) *node.Node {
 	return currentNode
 }
 
-func setupBlacklist() (*map[ethCommon.Address]struct{}, error) {
+func setupBlacklist() (map[ethCommon.Address]struct{}, error) {
```

Comment:
why you have a blacklist change in this PR? Please use a separate one for blacklist fix.

Reply:
okay, will remove blacklist change and add in followup pr